docs/conops.md

```diff
@@ -0,0 +1,146 @@
+
+For example, if a customer wants to ship a 10-kg cargo from San Francisco to San Jose departing April 10th, Arrow would present a list of options. So, the customer may see Elroy, Joby, and Zipline vehicles available for this particular itinerary but at different pricing points:
+
+*(Illustrative purpose only)*
+* Joby Cargo: $47.2
+* Lilium Cargo: $50.1
+* Zipline Cargo: $51.3
+
+All else being equal, the customer would naturally pick the Joby option because it is the cheapest. This behavior will incentivize OEMs to innovate in order to compete with other players on price.
+
+The innovation can come from hardware (advanced manufacturing, superior procurement management, economic design, etc), or from algorithm. Some well-funded players may not care about recouping CapEx, so they would develop algorithms to analyze the supply and demand of the market, pricing of other services, etc., to programmatically price their trips.
+
+#### Pricing as a Service
+Arrow may offer a set of APIs for OEMs to develop in-house algorithms, but it could also be a product offering.
+
+Since Arrow has the full-access to everything on the network, it may makes sense to offer a white-labeled pricing algorithm solution to OEM players, which 1) provides an additional source of revenue to Arrow, and 2) help these OEM players to narrow their focus to the things that they are good at -- making vehicles.
+
+We believe this business model creates a win-win situation where it releases the production and legal burdens from Arrow, and it serves as a direct-to-consumer channel to OEMs, allowing them to monetize on their vehicles with a shorter time-to-market than if they have to sell them through contracts.
+
+
+#### Downside
+While the marketplace business model bridges many gaps on the market amount different participants, the platform (Arrow) itself may under-perform as operating a marketplace is difficult and the success is largely a function [network effects](https://online.hbs.edu/blog/post/what-are-network-effects). 
+
+There essentially three parties in a marketplace ecosystem: the platform (Arrow), the service providers (OEMs or other fleet owner-operators), the customers (passengers, corporations, etc.).
+
+At the early stage, Arrow would have to make outreach efforts to both service providers and customers because a lack of either parties will fail the business. If Arrow were to pursue this business model, a well-thought-out go-to-market strategy is needed and the execution has to be near flawless.
+
+#### Further Discussions
+Key points to discuss:
+* Whether Arrow wants to pursue a marketplace model
+* If we go with the vertical integration model, what kind of operations are we capable of running commercially
+* Who and when to showcase our demo
+* Should we seek strategic investors
```

Comment:
Also mention grants and government subsidies being an alternate source of operating funds

There are "Foundations" (like Blender) that only subsist off of grants/donations, I think. Means they don't have to bend to the will of shareholders unless the donations are earmarked

Reply:
[caa132d](https://github.com/Arrow-air/svc-pricing/pull/3/commits/caa132d62cbff9e89dee9650443a7a8e39d9d466)